### PR TITLE
Add more exercises

### PR DESCRIPTION
**Pull request by sapegin**
Everything except unknown (???) and the last one with everything.

### CODE REVIEW

**Review by okonet — APPROVED** [review]
Very cool!

File: src/exercises/exercise05/Link.md [hunk 1]
```diff
@@ -1,4 +1,16 @@
...
 ```jsx
-<Link href="/">Normal link</Link>{' '}
-<Link href="/" shy>Shy link</Link>
+<Link href="/">Normal link</Link>
+```
+
+Shy link. Use for secondary content like footers or small copy.
+
+```jsx
+<p>
```
Comment (okonet) [inline src/exercises/exercise05/Link.md:10]: Should we use `Text` here instead? We're teaching not to use HTML elements :)

File: src/exercises/exercise09/Dogs.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Flex, Box } from 'grid-styled';
+import DogPhoto from '../../components/app/DogPhoto';
+
+const Dogs = ({ dogs }) => (
+	<div>{dogs.map(dog => <DogPhoto id={dog.id} breed={dog.name} />)}</div>
```
Comment (okonet) [inline src/exercises/exercise09/Dogs.js:7]: Is this a completed eyxcersice? Or what's the goal of the exercise?

File: src/exercises/exercise10/Readme.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Creating a subscription form
+
+## Getting started
+
+Run Styleguidist: `npm run exercise 10`.
+
+## Task
+
+Create a subscription form that consists for two components:
```
Comment (okonet) [inline src/exercises/exercise10/Readme.md]: `consists for` => `consists of`

**Review by okonet — CHANGES_REQUESTED** [review]

File: src/exercises/exercise05/Link.md (hunk 1 above)
Comment (sapegin) [reply src/exercises/exercise05/Link.md:10]: They don't share any styles now — link only has color ;-)

File: src/exercises/exercise09/Dogs.js (hunk 2 above)
Comment (sapegin) [reply src/exercises/exercise09/Dogs.js:7]: Check the readme ;-)

File: src/exercises/exercise10/Readme.md (hunk 3 above)
Comment (sapegin) [reply src/exercises/exercise10/Readme.md]: 👍

File: src/exercises/exercise05/Link.md (hunk 1 above)
Comment (okonet) [reply src/exercises/exercise05/Link.md:10]: Not sure I understand what you're saying. Is Text implemented at this point?

File: src/exercises/exercise09/Dogs.js (hunk 2 above)
Comment (okonet) [reply src/exercises/exercise09/Dogs.js:7]: Ah gotcha.

File: src/exercises/exercise05/Link.md (hunk 1 above)
Comment (sapegin) [reply src/exercises/exercise05/Link.md:10]: Final implementations don't share styles:

https://github.com/sapegin/component-driven-development/blob/master/src/components/core/Link.js
https://github.com/sapegin/component-driven-development/blob/master/src/components/core/Text.js

Comment (sapegin) [reply src/exercises/exercise05/Link.md:10]: But maybe they should, then we could use a Link outside Text, for example in navigation.

Comment (sapegin) [reply src/exercises/exercise05/Link.md:10]: Done.

**Review by okonet — APPROVED** [review]

**Pull request merged**